### PR TITLE
Newer Version: FujiApple.Trippy 0.8.0

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Holding off merged due to Winget v1.5 and higher version is not yet released. Converting to draft.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107665)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Do you know what happens to this pull request?

**Comment by BrandonWanHuanSheng** [comment]
What the pull request gives me Validation-Executable-Error?

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
error: the following required arguments were not provided:
  <TARGETS>...

Usage: trip.exe <TARGETS>...

For more information, try '--help'.
```

**Comment by stephengillie** [comment]
> What the pull request gives me Validation-Executable-Error?

It can't tell if the program launched successfully or not. So a human reviews.

**Comment by BrandonWanHuanSheng** [comment]
@mdanish-kh Do you know when the newer version winget-cli will release?

**Comment by mdanish-kh** [comment]
I'm not an MS employee so I can't say exactly but I recall it being mentioned that it's targeted for the end of this month.

**Comment by BrandonWanHuanSheng** [comment]
Marking pull request is ready to review.

File: manifests/f/FujiApple/Trippy/0.8.0/FujiApple.Trippy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=NVS0.CRLF.5-1-19041-3030.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FujiApple.Trippy
+PackageVersion: 0.8.0
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2023-05-15
+Installers:
```
Comment (stephengillie) [inline manifests/f/FujiApple/Trippy/0.8.0/FujiApple.Trippy.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie

**Comment by BrandonWanHuanSheng** [comment]
Hmm, what the pipeline is happening.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**